services/replicator/metadataReconciler.go

```diff
@@ -451,37 +474,179 @@ func (r *metadataReconciler) reconcileDestExtent(destUUID string, localExtents m
 				continue
 			}
 		} else {
-			if remoteExtentStatus != shared.ExtentStatus_OPEN && localExtentStatus == shared.ExtentStatus_OPEN {
-				r.logger.WithFields(bark.Fields{
-					common.TagDst:      common.FmtDst(destUUID),
-					common.TagExt:      common.FmtExt(remoteExtentUUID),
-					common.TagZoneName: common.FmtZoneName(remoteZone),
-				}).Info(`Inconsistent extent status`)
-				r.m3Client.UpdateGauge(metrics.ReplicatorReconcileScope, metrics.ReplicatorReconcileDestExtentInconsistentStatus, 1)
+			if remoteExtentStatus == shared.ExtentStatus_SEALED && localExtentStatus == shared.ExtentStatus_OPEN {
```

Comment:
added 'consumed' scenario. As discussed offline, since handleExtentDeletedOrMissingInRemote is a special/corner case, I feel like using a switch is not very suitable here.